design/adr/core/0001-MongoDB-Clearance.md

```diff
@@ -0,0 +1,58 @@
+# MongoDB Clearance
+
+<!--ts-->
+
+- [Status](#status)
+- [Context](#context)
+- [Proposed Design](#proposed-design)
+- [Decision](#decision)
+- [Consequences](#consequences)
+- [References](#references)
+
+<!--te-->
+
+## Status
+
+Proposed
+
+## Context  
+[docker-edgex-mongo](https://github.com/edgexfoundry/docker-edgex-mongo) has been developed to replace the old way of initializing mongo database (using [init_mongo.js](https://github.com/edgexfoundry/developer-scripts/blob/master/init_mongo.js)) and to introduce security features.  <br />
+Currently, docker-edgex-mongo is single purpose application, because it is responsible only for `initializing` all the data inside mongo. <br />
+It runs the mongo server, creates all the databases with the appropriate collections inside and create the relevant users.<br />
+Users credentials are taken from the secret store or from the configuration.toml file.  
+
+## Proposed Design 
+The proposal is another executable to be added inside docker-edgex-mongo, that will bring up `/purge` endpoint - responsible to purge all the data inside all the collections (thus, replacing the [clean_mongo.js](https://github.com/edgexfoundry/developer-scripts/blob/master/clean_mongo.js)) <br />
```

Comment:
Single purge endpoint to erase all data sounds like a security risk to me.  @tingyuz?

Reply:
> multi tenancy

Authentication and authorization...?